### PR TITLE
feat: cypress default configuration

### LINKED ISSUE #18: Test UI for Errors
Set up Cypress to automatically throw an error in CI if `static/index.html` throws any errors in the console on page load. 

This should catch most issues I think because our apps are so simple. 

No need to click buttons or anything yet.

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #18 

Note: it is now important that in Jest files imports are added explicitly, otherwise they will be interpreted as Mocha instead of Jest by the IDE (does not break the tests, just the auto completion).

### CODE REVIEW

File: cypress/e2e/main.cy.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+describe("Homepage tests", () => {
+  it("Console is cleared of errors and warnings", () => {
+    cy.visit("/", {
+      onBeforeLoad(win) {
+        cy.stub(win.console, "warn").as("consoleWarn");
+        cy.stub(win.console, "error").as("consoleError");
+      },
+    });
+    cy.get("@consoleWarn").should("not.be.called");
```
Comment (0x4007) [inline cypress/e2e/main.cy.ts]: Warn is probably okay but console errors and thrown errors probably not.

**Review by 0x4007 — CHANGES_REQUESTED** [review]

**Comment by 0x4007** [comment]
Tag me or rndquu when you need this merged. Looks like our permissions system for generating permits is still functional. 

We are `admins` or `billing_managers` according to the org settings so we have authority to generate permits.

File: cypress/e2e/main.cy.ts (hunk 1 above)
Comment (gentlementlegen) [reply cypress/e2e/main.cy.ts]: Gotcha will just check for errors then.

**Comment by 0x4007** [comment]
By the way tagging means like this @FernandVEYRIER because I know that I get push notifications for these.

**Comment by 0x4007** [comment]
Hey @FernandVEYRIER please adjust this to only post comments on pulls instead. 

https://github.com/ubiquity/series-a/commit/fb63aa17417ee1eb05ba9b1c7ad4c9b4db657b5a#commitcomment-139370792

**Pull request merged**